src/main.rs

```diff
@@ -110,7 +110,14 @@ fn main() -> Result<()> {
                 .long("ascii_language")
                 .takes_value(true)
                 .default_value("")
-                .help("Overrides showing the dominant language ascii logo"),
+                .help(&format!(
+                    "Overrides showing the dominant language ascii logo. Possible values: [{}]",
+                    Language::iter()
+                        .filter(|language| *language != Language::Unknown)
+                        .map(|language| language.to_string().to_lowercase())
+                        .collect::<Vec<String>>()
+                        .join(", ")
```

Comment:
You can actually [set possible values](https://docs.rs/clap/2.33.0/clap/struct.Arg.html#method.possible_values) for a `clap::Arg`. This will not only show in the help message \(assuming you don't [hide them](https://docs.rs/clap/2.33.0/clap/struct.Arg.html#method.hide_possible_values)\), but will also make `clap` ensure that *only* the possible values are used. e.g. If someone runs `onefetch -a French`, they will receive an error message saying that's an unexpected input, and prompting them to view the help message.